lib/ae/base/ae_template.py

```diff
@@ -10,17 +12,35 @@ def __init__(self, ufeSceneItem):
         self.attrS = ufe.Attributes.attributes(ufeSceneItem)
         super(AEBaseTemplate, self).__init__(ufeSceneItem)
 
-    def createSection(self, layoutName, attrList):
+    def createSection(self, layoutName, attrList, collapse=False):
         # We create the section named "layoutName" if at least one
         # of the attributes from the input list exists.
         for attr in attrList:
             if self.attrS.hasAttribute(attr):
-                with ufeAeTemplate.Layout(self, layoutName):
+                with ufeAeTemplate.Layout(self, layoutName, collapse):
                     # Note: addControls will silently skip any attributes
                     #       which does not exist.
                     self.addControls(attrList)
                 return
 
+    def createTransformAttributesSection(self):
+        # Get the xformOp order and add those attributes (in order)
+        # followed by the xformOp order attribute.
+        prim = mayaUsd.ufe.getPrimFromRawItem(self.item.getRawAddress())
+        geomX = UsdGeom.Xformable(prim)
+        xformOps = geomX.GetOrderedXformOps()
+        xformOpOrderNames = [op.GetOpName() for op in xformOps]
+        xformOpOrderNames.append(UsdGeom.Tokens.xformOpOrder)
+        # Don't use createSection because we want a sub-section for Unused.
+        with ufeAeTemplate.Layout(self, 'Transform Attributes'):
+            self.addControls(xformOpOrderNames)
+
+            # Get the remainder of the xformOps and add them in an Unused section.
+            allAttrs = self.attrS.attributeNames
+            xformOpUnusedNames = fnmatch.filter(allAttrs, 'xformOp:*')
+            xformOpUnusedNames = [ele for ele in xformOpUnusedNames if ele not in xformOpOrderNames]
+            self.createSection('Unused Transform Attributes', xformOpUnusedNames, collapse=True)
```

Comment:
Then get (into a list) all the attribute names that match the wildcard string "xformOp:*" and remove from that list the attributes from the ordered list above. Then we create an "Unused" sub-section with those.